lib/tasks/api_keys.rake

```diff
@@ -0,0 +1,22 @@
+namespace :api_keys do
+  desc "Migrate user api keys to ApiKey model"
+  task migrate: :environment do
+    users = User.all
+
+    total = users.count
+    i = 0
+    puts "Total: #{total}"
+    users.find_each do |user|
+      hashed_key = Digest::SHA256.hexdigest(user.api_key)
+      scopes_hash = ApiKey::API_SCOPES.each_with_object({}) { |k, h| h[k] = true }
+
+      api_key = user.api_keys.create(scopes_hash.merge(hashed_key: hashed_key, name: "legacy-key"))
```

Comment:
Will there be a validation problem here due to the show_dashboard scope not being exclusive?

Reply:
yes, will disable validation here.